azplugins/CMakeLists.txt

```diff
@@ -120,14 +120,21 @@ list(APPEND py_files
 # mpcd c++ files
 list(APPEND _${COMPONENT_NAME}_sources
     MPCDReversePerturbationFlow.cc
+    MPCDSinusoidalChannelFiller.cc
+    MPCDSinusoidalExpansionConstrictionFiller.cc
     )
 # mpcd c++/cu cuda files
 if(ENABLE_CUDA)
 list(APPEND _${COMPONENT_NAME}_sources
     MPCDReversePerturbationFlowGPU.cc
+    MPCDSinusoidalChannelFillerGPU.cc
+    MPCDSinusoidalExpansionConstrictionFillerGPU.cc
     )
 list(APPEND _${COMPONENT_NAME}_cu_sources
+    MPCDStreamingMethod.cu
```

Comment:
It failed spectacularly... so no way of accidentally running it without noticing that it is wrong. Anyway, I agree -- we should remove this. The filler are a bit more involved so I would want to keep them in case we want to use mpcd on GPUs at some point in the future.

Reply:
I'm going to remove the file `MPCDStreamingMethod.cu` altogether.